src/main/resources/bootstrap.yaml

```diff
@@ -0,0 +1,6 @@
+spring:
+  cloud:
+    propertiesvolume:
+      paths: /mnt/secrets/ccd
+      aliases:
```

Comment:
Shouldn't this be mounted  in the chart values.template.yaml?

Reply:
no, in values.template.yaml it uses the postgres chart, in AAT it uses another DB so only needed there

Reply:
I see, but are you sure you are declaring this keyvault flexvolume mount? I can't find it neither in values.template.yaml nor in values.aat.template.yaml.

Reply:
it'll go into cnp-flux-config only